src/template-generator.js

```diff
@@ -0,0 +1,28 @@
+const HandleBars = require("handlebars");
+
+module.exports = (customerName, budget) => {
```

Comment:
I think that full payload can be received as parameter to open this function to extension and avoid to every new parameter be added we needs to modify this function.

Reply:
I 'll think in the best way to do that here

Reply:
I removed handlebars and used just string interpolation. Passed the full payload as argument.